lib/models/config.ts

```diff
@@ -127,6 +127,7 @@ const getConfigModel = function (
 		 * @function
 		 * @memberof balena.models.config
 		 *
+		 * @deprecated
```

Comment:
reminder to do something like:
```suggestion
		 * @deprecated use in favor of balena.models.deviceType.getAll
```